RELEASENOTES.md

```diff
@@ -8,6 +8,27 @@ Note that when using the preview version of AL-Go for GitHub, we recommend you U
 - App artifacts for version 'latest' are now fetched from the latest CICD run that completed and successfully built all the projects for the corresponding branch.
 - Issue 824 Utilize `useCompilerFolder` setting when creating an development environment for an AL-Go project.
 
+### New Settings
+
+- `alDoc` : Structure with properties for the ALDoc reference document generation
+  - **ContinuousDeployment** = Determines if reference documentation will be deployed continuously as part of CI/CD. You can run the **Deploy Reference Documentation** workflow to deploy manually or on a schedule. (Default false)
+  - **DeployToGitHubPages** = Determines whether or not the reference documentation site should be deployed to GitHub Pages for the repository. In order to deploy to GitHub Pages, GitHub Pages must be enabled and set to GitHub Actuibs. (Default true)
+  - **MaxReleases** = Maximum number of releases to include in the reference documentation. (Default 3)
+  - **useProjectsAsFolders** = Determines whether projects in multi-project repositories are used as folders in reference documentation
```

Comment:
Why is this setting's name is camel-cased and the others are pascal-cased?

Reply:
typo